docs/_static/support.inc

```diff
@@ -3,7 +3,7 @@
 Support / Feedback
 ==================
 
-The packages included in **nimi-python** package are supported by NI. For support, open
+The packages included in the **nimi-python** repository are supported by NI. For support, open
```

Comment:
I think we should reword.

For supported specific to the Python API, go to <issues and forum page in GitHub>.
For support with the driver runtime, go to <ni.com support forum>.

Reply:
I reworded it. This file is pulled in by both the READMEs shown on the PyPI page and the index for the readthedocs documentation. Let me know what you think.